src/sync/incomingSyncUtils.js

```diff
@@ -373,7 +373,7 @@ export const createOrUpdateRecord = (database, settings, recordType, record) =>
         name: record.description,
         note: record.note,
         isProgram: parseBoolean(record.isProgram),
-        programSettings: JSON.stringify(record.programSettings),
+        programSettings: record.programSettings ? JSON.stringify(record.programSettings) : '',
```

Comment:
As per comments up the track, `record.programSettings && record.programSettings.replace(/\\/g, '')`

Reply:
done as above